### PR TITLE
failing test

### PR DESCRIPTION
**Pull request by sphaso**
#3658

### CODE REVIEW

File: tests/integration/test.replication.js [hunk 1]
```diff
@@ -3609,6 +3609,37 @@ adapters.forEach(function (adapters) {
       });
     });
 
+    it('#3658 replication with ddoc filter, update doc', function() {
+      var db = new PouchDB(dbs.name);
+      var remote = new PouchDB(dbs.remote);
+
+	      return remote.bulkDocs([{
```
Comment (nolanlawson) [inline tests/integration/test.replication.js]: It seems the bug is that you are putting the design doc in the remote database. If you are replicating *from* the local database, then that is where the ddoc should go. :)

**Comment by sphaso** [comment]
I'll try to remove the filter instruction in the local -> remote replication. 
So when I use "sync", does the ddoc needs to be in both local and remote DB?

**Comment by nolanlawson** [comment]
> So when I use "sync", does the ddoc needs to be in both local and remote DB?

Yes.

**Comment by sphaso** [comment]
This is very counterintuitive (at least to me). I guess I could change the ddoc filter remote-side to have it replicated locally ( like "conditions... || doc.id.startsWith("_design"), then I could use "sync" without worrying about it.
I removed the "filter" option from local -> remote replicate but the test is failing anyway :/

**Comment by daleharvey** [comment]
Filtered replication got fixed and looks like this got cleared up, closing, Thanks a lot

**Pull request closed without merging**